packages/cli/src/list.js

```diff
@@ -1,13 +1,16 @@
 import chalk from 'chalk'
 import NuxtCommand from './command'
+import { getLocalCommands, loadLocalCommand } from './local'
```

Comment:
An small naming suggestion, we can call these commands `builtIn` or `core` :)

Reply:
@pi0 not really, `NuxtCommand` represents the built-in ones, `local` has commands local _to the project_ only. In my othe branch, I've evolved this into NuxtCommand subclasses, `LocalNuxtCommand` (for locally defined commands) and `ExternalNuxtCommand` (for commands defined inside NPM packages).